src/audio/google/google_rtc_audio_processing.c

```diff
@@ -8,6 +8,7 @@
 #include <ipc/stream.h>
 #include <ipc/topology.h>
 #include <ipc4/aec.h>
+#include <sof/audio/module_adapter/module/generic.h>
```

Comment:
if this patch have multiple purpose(from commit message), I would suggest split out 1 or 2 patches.

Reply:
yes please